README.md

```diff
@@ -157,6 +157,12 @@ This instance also stores and retrieves data in the Secure Enclave, but does not
 
 **In order for your customers not to receive a prompt that your app does not yet support Face ID, you must set a value for the Privacy - Face ID Usage Description [(NSFaceIDUsageDescription)](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html#//apple_ref/doc/uid/TP40009251-SW75) key in your app’s Info.plist.**
 
+### Thread Safety
+
+Valet is built to be thread safe: it is possible to use a Valet instance on any queue or thread. Valet instances ensure that code that talks to the Keychain is atomic – it is impossible to corrupt data in Valet by reading and writing on multiple queues simultaneously.
+
+Because Keychain items are stored on disk, it is recommended that you interact with your Valet instance on a background queue to prevent blocking the main queue.
```

Comment:
At first glance this appears to contradict the previous paragraph. Maybe something more like:

> However, accessing a Valet instance from the main queue can result in blocking behavior like UI updates. We recommend using a background queue to access your Valet instance to prevent this from occurring.

Reply:
That's way better! I've updated the language. Let me know what you think 🙂